File: output/LLaMa/avatar/Java/Python/atcoder_ABC153_A.py
import sys

def atcoder_ABC153_A():
    h = int(input().split()[0])
    a = int(input().split()[1])
    num = 0
    for i in range(1, h + 1):
        h = h - a
        num += 1
    return num

print(atcoder_ABC153_A())

File: output/LLaMa/avatar/Python/Java/atcoder_ABC153_A.java
import java.util.Scanner;

public class atcoder_ABC153_A {
    public static void main(String[] args) {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Enter h and a separated by space: ");
        String[] input = scanner.nextLine().split(" ");
        int h = Integer.parseInt(input[0]);
        int a = Integer.parseInt(input[1]);
        int sho = h / a;
        int am = h % a;
        if (am != 0) {
            sho++;
        }
        System.out.println(sho);
    }
}